packages/cypress/migrations.json

```
@@ -79,6 +73,18 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "16.5.0": {
+      "version": "16.5.0-beta.1",
```

Not a big deal, but I think we start with `beta.0` and it hasn't been published yet.
```suggestion
      "version": "16.5.0-beta.0",
```

---

oops. typo'd that. Thanks!